docroot/sites/admissions.uiowa.edu/modules/admissions_core/es6/slideshow.es6

```diff
@@ -76,3 +79,12 @@ document.addEventListener("DOMContentLoaded", function() {
     return styleString;
   }
 });
+
+// https://stackoverflow.com/a/12646864
+/* Randomize array in-place using Durstenfeld shuffle algorithm */
+function shuffleArray(array) {
+  for (let i = array.length - 2; i > 0; i--) {
```

Comment:
Is there a reason I'm missing that it's `i = array.length - 2` and not `i = array.length - 1`?

Reply:
Nope! I think I was experiencing a bug for other reasons but forgot to change it back. Good check!